lib/iiif_print.rb

```diff
@@ -161,6 +161,12 @@ def self.allinson_flex_fields_for(_work, fields: allinson_flex_fields)
         options: options
       )
     end
+    # Adding collection metadata for display in the UV metadata pane
+    if fields.where(name: 'collection').empty?
```

Comment:
Favor `fields.exists?(name: 'collection')`. Is this column name "magical" or configurable?

https://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-exists-3F